qa-test/build.rs

```diff
@@ -2,8 +2,12 @@ fn main() {
     // Allow building QA tests in CI in debug mode
     println!("cargo:rustc-check-cfg=cfg(is_not_release)");
     println!("cargo:rerun-if-env-changed=CI");
-    #[cfg(debug_assertions)]
+    println!("cargo:rerun-if-env-changed=CI");
```

Comment:
Do we need two of these?

Reply:
Just in case ... better safe than sorry 😄